crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -2206,6 +2208,8 @@ mod tests {
     async fn test_setting_the_latest_event_doesnt_cause_a_room_info_update() {
         // Given a room,
 
+        use ruma::user_id;
```

Comment:
Is this used? if so, can you put it with the regular `use` statements at the top of this module instead?

Reply:
Cool that there are guidelines for commit and pr names/messages.